applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -604,6 +656,13 @@ The application supports the following configurations:
      - Debug version of the application.
 
        Enables additional features for verifying the application behavior, such as logs.
+   * - Debug
+     - :file:`prj_nrf70ek.conf`
+     - ``nrf70ek``
+     - ``nrf5340dk``
+     - Debug version of the application.
+
+       Enables Matter-Bridge to work with Wi-Fi.
```

Comment:
Aligning more with the Thread and Wi-Fi switching entry for the [Matter door lock configurations](https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/samples/matter/lock/README.html#matter_door_lock_custom_configurations)
![image](https://github.com/user-attachments/assets/d7c252f8-1f79-45a6-a58f-477bf64ba8d8)
```suggestion
   * - Matter bridge over Wi-Fi
     - :file:`prj_nrf70ek.conf`
     - ``nrf70ek``
     - nRF5340 DK with the nRF7002 EK shield attached
     - Debug version of the application with Matter over Wi-Fi enabled.
```
Should also be moved to be the last entry in the list, I think. We don't really have much precedent beyond the Door lock sample, which places the the Debug and Release configurations first.

Is this configuration only used when working with the nRF5340 DK and the shield? If so, that might need to be added to the description.